bundles/org.openhab.binding.froniuswattpilot/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,210 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="froniuswattpilot"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Thing Types -->
+	<thing-type id="wattpilot">
+		<label>Fronius Wattpilot</label>
+		<description>Fronius Wattpilot wallbox</description>
+		<semantic-equipment-tag>EVSE</semantic-equipment-tag>
+
+		<channel-groups>
+			<channel-group id="control" typeId="control"/>
+			<channel-group id="status" typeId="status"/>
+			<channel-group id="metrics" typeId="metrics"/>
+		</channel-groups>
+
+		<properties>
+			<property name="serialNumber"/>
+			<property name="firmwareVersion"/>
+			<property name="thingTypeVersion">1</property>
+		</properties>
+
+		<representation-property>hostname</representation-property>
+
+		<config-description>
+			<parameter name="hostname" type="text" required="true">
+				<context>network-address</context>
+				<label>Hostname</label>
+				<description>Hostname or IP address of the device</description>
+			</parameter>
+			<parameter name="password" type="text" required="true">
+				<context>password</context>
+				<label>Password</label>
+				<description>Password to access the device (the same password as required by the Solar.Wattpilot app)</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<!-- Channel Group Types -->
+	<channel-group-type id="control">
+		<label>Charging Control</label>
+		<channels>
+			<channel id="charging-allowed" typeId="charging-allowed"/>
+			<channel id="charging-mode" typeId="charging-mode"/>
+			<channel id="charging-current" typeId="charging-current"/>
+			<channel id="pv-surplus-threshold" typeId="pv-surplus-threshold"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="status">
+		<label>Charging Status</label>
+		<channels>
+			<channel id="charging-state" typeId="charging-state"/>
+			<channel id="charging-possible" typeId="charging-possible"/>
+			<channel id="single-phase" typeId="charging-single-phase"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="metrics">
+		<label>Charging Metrics</label>
+		<channels>
+			<channel id="power" typeId="system.electric-power">
+				<label>Total Power</label>
+			</channel>
+			<channel id="energy-session" typeId="system.electric-energy">
+				<label>Charged Energy</label>
+				<description>Energy charged in the current/last session</description>
+			</channel>
+			<channel id="energy-total" typeId="system.electric-energy">
+				<label>Total Charged Energy</label>
+				<description>Energy charged in total</description>
+			</channel>
+			<channel id="l1-power" typeId="system.electric-power">
+				<label>Phase 1 Power</label>
```

Comment:
You might want to override description as well, otherwise it will default to "Current electric power" (for all channels referring to `system.electric-power`).